autokeras/auto/auto_model.py

```diff
@@ -81,6 +91,18 @@ def _get_metrics(self):
             metrics += metrics_list
         return metrics
 
+    def _compiled(self, hp, model):
+        optimizer = hp.Choice('optimizer',
```

Comment:
Why are these choices hard-coded?

Reply:
Do you mean the search space is small? I think this will be changed the later to be more flexible. Currently, it is just a selection among these optimizers.